src/rock/models/finances/__tests__/__snapshots__/mutations.js.snap

```diff
@@ -13,34 +13,35 @@ Array [
     start_date: String,
     end_date: String,
     personId: Int,
-    gateway: String = \\"${process.env.NMI_GATEWAY}\\",
+    gateway: String = \\"undefined\\",
```

Comment:
Not sure where this could change...

Reply:
This should probably be returning the `process.env.NMI_GATEWAY` right? Not undefined.

Reply:
I'm trying to figure out why we even have all of these instances of `${process.env.NMI_GATEWAY}` anyways.